plugins/pipelines/lib/samson_pipelines/samson_plugin.rb

```diff
@@ -14,12 +14,13 @@ def start_pipelined_stages(job, success, output)
 
     private
 
-    def deploy_to_stage(stage, template_deploy, output)
-      deploy_service = DeployService.new(template_deploy.user)
+    def deploy_to_stage(stage, previous_deploy, output)
```

Comment:
not really an obvious 2nd argument, also should be named triggering_deploy and not previous ?

Reply:
nvm this was just a rename :D